CMakeLists.txt

```diff
@@ -4,7 +4,7 @@
 #                          Daniel Nachbaur <daniel.nachbaur@epfl.ch>
 
 cmake_minimum_required(VERSION 3.1 FATAL_ERROR)
-project(Deflect VERSION 0.13.1)
+project(Deflect VERSION 0.14.0)
```

Comment:
I suspect the split of Stream in Observer + Stream to break the ABI as well, no?